libherokubuildpack/Cargo.toml

```diff
@@ -45,6 +49,14 @@ termcolor = { version = "1.4.0", optional = true }
 thiserror = { version = "1.0.50", optional = true }
 toml = { workspace = true, optional = true }
 ureq = { version = "2.9.1", default-features = false, features = ["tls"], optional = true }
+lazy_static = { version = "1", optional = true }
+regex = { version = "1", optional = true }
```

Comment:
I'm pretty against adding a dependency on the `regex` crate here - since:
- it's a very large crate given unicode support (particularly for smaller buildpacks), and whilst some other parts of libcnb use `fancy_regex` (a) I have plans to remove those, (b) it's a different crate so using both crates causes overlap since they don't fully share the same deps
- looking at the two usages of the `regex` crate in the logger, it seems a non-regex approach would both be possible and likely more appropriate/simpler/quicker